### ISSUE TITLE
NTR, pulmo: lung shrinking

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#6437]
**Preferred term label:**
shrinking of the lung

**Synonyms**
Shrinking lung syndrome, “vanishing” lung syndrome, small lungs

**Definition (free text, please give PubMed ID)**
reduced lung volumes resulting in restrictive
ventilatory defect and a preserved carbon monoxide transfercoefficient.

Comment: Chest x-ray often shows small but clear lungs. It is mostly presented in combination with diaphragmatic elevation and occasional basal atelectasis

**Parent term (use hpo.jax.org/app)**
Aplasia/Hypoplasia of the lungs HP:0006703

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:536   (Disorder)   Systemic lupus erythematosus
Usual interstitial pneumonia
     PMID: 24836585 DOI: 10.1177/0961203314536248 

**Your nano-attribution (ORCID)**
lmu: chrapp, @mgriese, @kknoflach

**Comment by pnrobinson** [comment]
This seems to be a disease with multiple diagnostic criteria
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5370827/
Is there a phenotypic feature that we can denote by shrinking lung? "Reduced lung volume" can be interpreted as "lower than normal" (today) or "reduced over time" (today less than last year). It is unclear to me from the article above what is included in the definition, although only the latter makes sense.

**Comment by pnrobinson** [comment]
Apparently,  "vanishing lung" is not the same thing -- https://radiopaedia.org/articles/idiopathic-giant-bullous-emphysema?lang=us

**Comment by pnrobinson** [comment]
Reading case reports, there is not extensive evidence of actual shrinking. All  cases have restrictive defects. Most recent cases have just imaging with pleural effusion, basal atelectasis, diaphragm elevation. Some older cases (https://pubmed.ncbi.nlm.nih.gov/14278816/) have necropsy reports with fibrous pleural adhesions, consolidations, etc. One of the cases mentions small lung but with no quantification.
I think that it is probably better to qualify Shrinking lung syndrome as a disease -- lupus, but also some other CT diseases such as Sjogren can lead to it. We can the annotate it with the HPO terms as shown above. @nicolevasilevsky @cmungall @mellybelly  this is an interesting pattern for Mondo -- gray zone between disease and phenotypic feature in common disease, we should discuss rules for curation and document them somewhere.

fibrous pleural adhesions and small lungs with some basal
collapse.

**Comment by mgriese** [comment]
A)
Vanishing Lung (VLS) is also referred to as idiopathic giant bullous emphysema

Definition from "An Idiopathic Bullous Emphysema Mimicking Pneumothorax. Cureus 12(8): e9596. doi:10.7759/cureus.9596"
...... a rare manifestation of chronic obstructive pulmonary disease (COPD). Middle-aged tobacco smokers, younger marijuana users, and those with alpha-1-antitrypsin deficiency may especially be affected. The clinical and radiographic findings of VLS may initially be misinterpreted as spontaneous pneumothorax. High-resolution CT is the diagnostic imaging modality of choice in these patients and can help to differentiate VLS from pneumothorax. Such imaging also helps guide appropriate management. Management of VLS ranges from a conservative to a surgical approach depending upon patients’ comorbidities and candidacy for surgical resection. We present a case of a 64-year-old man with frequent hospitalizations for COPD exacerbation admitted with worsening shortness of breath and was found to have giant bullae mimicking a pneumothorax on the initial presentation.

is completely different from

B) 
Shrinking lung syndrome 
Definition and expalnation from PMID: 21686610
SLS, coined by Hoffbrand and Beck in 1965, is a rare pulmonary manifestation of SLE.1 The pathophysiology of SLS is unclear; a myopathic process affecting the diaphragm or intercostal muscles is thought to be the primary abnormality.2 Exercise intolerance over weeks to months is the common mode of presentation and is seen months to years after SLE is diagnosed. The respiratory dysfunction appears to be independent of the clinical course, duration and lupus antibodies (ANA and anti-DNA).3 Characteristic diagnostic findings include elevated diaphragms on imaging and restrictive lung volumes on pulmonary function tests.2 The absence of other pathological conditions is evidenced in most reports by normal V/Q scans, bronchoscopy and lung biopsies. Prognosis is generally good and most patients improve on increasing the dose of steroids aimed at the subclinical respiratory myositis-like process.3

**Comment by pnrobinson** [comment]
I would suggest that both of the above items are best characterized by diseases in MONDO @nicolevasilevsky @cmungall  can we add these terms in the next Mondo meeting. 
@mgriese I would say that VLS/ idiopathic giant bullous emphysema should be a subclass of pulmonary emphysema (https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0004849) and
Shrinking lung syndrome  -- not sure how to classify, perhaps just lung disease for now (https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0005275&viewMode=PreferredRoots&siblings=false)

I will transfer this to the mondo tracker
